src/main/java/org/ays/auth/controller/AysUserController.java

```diff
@@ -131,4 +132,23 @@ public AysResponse<Void> update(@PathVariable @UUID final String id,
         return AysResponse.SUCCESS;
     }
 
+
+    /**
+     * PATCH /user/{id}/activate : Activates a user account with the given ID.
+     * <p>
+     * This endpoint is protected and requires the caller to have the authority
+     * 'user:update'. The user ID must be a valid UUID.
+     * </p>
+     *
+     * @param id The UUID of the user to be activated.
+     * @return An {@link AysResponse} indicating the success of the operation.
+     */
+    @PatchMapping("user/{id}/activate")
+    @PreAuthorize("hasAnyAuthority('user:update')")
+    public AysResponse<Void> activate(@PathVariable @UUID final String id){
```

Comment:
kod formatlanmamış gibi duruyor
```suggestion
    public AysResponse<Void> activate(@PathVariable @UUID final String id) {
```